### PR TITLE
[doc] Update Mac build instructions

### PR DESCRIPTION
**Pull request by jimmy-huang**
Signed-off-by: Jimmy Huang <jimmy.huang@intel.com>

### CODE REVIEW

**Comment by jimmy-huang** [comment]
@haoxli please review

File: README.md [hunk 1]
```diff
@@ -474,6 +479,12 @@ After installing crosstool-ng, create and mount the image using our script:
 osxmountzephyr.sh
```
Comment (haoxli) [inline README.md]: >  You'll need to install Homebrew first so you can install crosstool-ng. Follow the Zephyr instructions [here](https://www.zephyrproject.org/doc/getting_started/installation_mac.html)

We have installed Homebrew, so here we can update to
> Install crosstool-ng following the Zephyr instructions [here](https://www.zephyrproject.org/doc/getting_started/installation_mac.html)

Comment (haoxli) [inline README.md]: I think `osxmountzephyr.sh` should have a hyperlink, or post source code here.

**Comment by haoxli** [comment]
LGTM.

**Comment by grgustaf** [comment]
+1

**Pull request merged**